### ISSUE TITLE
Keystone block & Gold fire dont render properly in world

### ISSUE THREAD

**Issue by freopt** [Andromander/Gaia-Dimension#37]
The keystone block is black and the fire triggers a ccl exception.

**Comment by cipherzerox** [comment]
Do you have CTM installed?

**Comment by Andromander** [comment]
CTM is required, however I know of a fix for non-CTM userd I might push later. 

As for the fire...strange that I haven't come across this crash. Can you post a log?

**Comment by freopt** [comment]
OK so ctm fixed it thanks. I got this crash when a little while after i placed it this time. https://paste.dimdev.org/enucahiqeg.mccrash Here is the full log on the same launch that I crashed: https://paste.dimdev.org/lovolefiba.md

**Comment by freopt** [comment]
For context: when I reported, the fire didn't crash, it just didn't render and it said something in chat

**Comment by Andromander** [comment]
Hmm, interesting, but at least it does give me something to work with. From what I can get at, entities don't seem to respond nicely to the fire, especially Rabbits.

**Comment by Andromander** [comment]
So, regarding the console spam from CodeChickenLib, I have made some significant changes to how Gold Fire renders. However, I'm not sure if this will fix it since I'm not sure if it was something the block was doing itself, or CCL was altering. As for the crash itself, I tried to replicate it myself after implementing changes and haven't replicated that one, too.

Try [this release](https://github.com/Andromander/Gaia-Dimension/releases/tag/untagged-63581b55105dac06a707) if you want to check it as well.
